readme.md

```diff
@@ -313,6 +313,7 @@
 - [Vapor](https://github.com/Cellane/awesome-vapor#readme) - Server-side development in Swift.
 - [Dash](https://github.com/ucg8j/awesome-dash#readme) - Python web app framework.
 - [FastAPI](https://github.com/mjhea0/awesome-fastapi#readme) - Python web app framework.
+- [IAM](https://github.com/kdeldycke/awesome-iam#readme) - Identity and Access Management.
```

Comment:
The description needs to elaborate more on the topic. And should not be title-case.

Reply:
I have several proposals:
* Expand the acronym in the title: `Identity and Access Management - <new description>.`
* Keep it that way: `IAM - <new description>.`

Which one you prefer?

Reply:
As for the new description, I pushed it in: https://github.com/sindresorhus/awesome/pull/1916/commits/43ac52c294bd44bbd229f362dec855f39717a2fa

Reply:
I choose to keep the `IAM` acronym as I found a couple of non-expended domain-specific acronyms in the awesome list (`IoT`, `CTF`, `GDPR`, `LIDAR`, `OKR`, ...), so I guess it is a reasonable choice.